tools/accuracy_checker/accuracy_checker/annotation_converters/__init__.py

```diff
@@ -61,5 +62,6 @@
     'BratsNumpyConverter',
     'Cifar10FormatConverter',
     'MNISTCSVFormatConverter',
-    'WMTConverter'
+    'WMTConverter',
+    'common_semantic_segmentation'
```

Comment:
is there any reason to have different naming convention?

Reply:
fixed, thanks